salt/cloud/clouds/linode.py

```diff
@@ -993,6 +993,58 @@ def get_password(vm_):
     )
 
 
+def _decode_linode_plan_label(label):
+    """
+    Attempts to decode a user-supplied Linode plan label
+    into the format in Linode API output
+
+    label
+        The label, or name, of the plan to decode.
+
+    Example:
+        `Linode 2048` will decode to `Linode 2GB`
+    """
+    sizes = avail_sizes()
+
+    if label not in sizes:
+        if "GB" in label:
+            raise SaltCloudException(
+                'Invalid Linode plan ({}) specified - call avail_sizes() for all available options'.format(label)
+            )
+        else:
+            plan = label.split()
+
+            if len(plan) != 2:
+                raise SaltCloudException(
+                    'Invalid Linode plan ({}) specified - call avail_sizes() for all available options'.format(label)
+                )
+
+            plan_type = plan[0]
+            try:
+                plan_size = int(plan[1])
+            except Exception as e:
```

Comment:
Can you change this to a more specific exception? We typically try to stay away from broad exceptions like this. You also don't need the `as e` here since you're not doing anything with it.